src/pages/flock-2025-workshop.md

```diff
@@ -0,0 +1,73 @@
+# Smarter package maintenance with Packit (Flock 2025)
+
+## 🎯 Goal
+
+The main goal of this workshop is to **set up a complete Fedora release pipeline** for your package using the following Packit jobs:
+
+- `pull_from_upstream` (alternatively `propose_downstream`, if you already use Packit in upstream)
+- `koji_build`
+- `bodhi_update`
+
+By the end of this session, you should have a working configuration that automates these steps in the Fedora package release lifecycle.
+
+---
+
+## 🛠️ Instructions
+
+### Prerequisites
+
+- Ensure you have a **Fedora package** you maintain/co-maintain. Ideally, pick one that has a **pending upstream release**, as this allows trying things out easily.
+
+### Setup
+
+For detailed setup instructions, you can follow [the documentation](https://packit.dev/docs/fedora-releases-guide/dist-git-onboarding#setup). You can find a brief summary below:
+
+#### Create a working branch
```

Comment:
Maybe mention also a fork to not push the branch to the `rpms/whatever`? You can push, you cannot delete afterwards, I already have one branch like that :/